conf/routes

```diff
@@ -20,7 +20,6 @@ POST        /digital/country                 controllers.DigitalPack.handleCount
 # Checkout
 GET         /checkout                        controllers.Checkout.renderCheckout
 POST        /checkout                        controllers.Checkout.handleCheckout
-GET         /checkout/thankyou               controllers.Checkout.thankyou
```

Comment:
Had a quick chat about this with @blackyjnz and @davidmcdowell, and they don't think there is a business requirement for a bookmarkable thank you page. Happy to implement this if you guys think this would add value from the technical standpoint. @rtyley @davidrapson @tudorraul any opinions?